### ISSUE TITLE
InputType problem when reading pre-trained Keras model

### ISSUE THREAD

**Issue by bagolysz** [deeplearning4j/deeplearning4j#4846]
I am trying to load a Keras model from json config file. Maybe there is a problem recognizing the Input Type, as my data is formatted as **(d=1, h=48, w=48)**, so channels go first, but when reading from json, it tries to put channels last: **Input type = InputTypeConvolutional(h=1,w=48,d=48)**

This line of code 
`ComputationGraphConfiguration conf = KerasModelImport.importKerasModelConfiguration(MODEL_PATH, false);`
is giving the following exception:

`Exception in thread "main" org.deeplearning4j.exception.DL4JInvalidConfigException: Invalid configuration for layer (idx=-1, name=conv2d_8, type=ConvolutionLayer) for height dimension:  Invalid input configuration for kernel height. Require 0 < kH <= inHeight + 2*padH; got (kH=3, inHeight=1, padH=0)
Input type = InputTypeConvolutional(h=1,w=48,d=48), kernel = [3, 3], strides = [1, 1], padding = [0, 0], layer size (output depth) = 32, convolution mode = Same
	at org.deeplearning4j.nn.conf.layers.InputTypeUtil.getOutputTypeCnnLayers(InputTypeUtil.java:117)
	at org.deeplearning4j.nn.conf.layers.ConvolutionLayer.getOutputType(ConvolutionLayer.java:156)
	at org.deeplearning4j.nn.modelimport.keras.layers.convolutional.KerasConvolution2D.getOutputType(KerasConvolution2D.java:149)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.inferOutputTypes(KerasModel.java:261)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:164)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:91)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelBuilder.buildModel(KerasModelBuilder.java:123)
	at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasModelConfiguration(KerasModelImport.java:256)
	at MainClass.main(MainClass.java:21)`

The json model is [input_test.json.tar.gz](https://github.com/deeplearning4j/deeplearning4j/files/1848118/input_test.json.tar.gz)

Using:
* Deeplearning4j version: 0.9.2-SNAPSHOT
* Keras 2.1.2

**Comment by maxpumperla** [comment]
@bagolysz I'm curious, what happened?

**Comment by bagolysz** [comment]
@maxpumperla I tried to create and save the same model from Keras using "channels_last" configuration setup. This way I could import it to Java without problems. So, I thought I will go with it that way.
